tools/dist_train.sh

```diff
@@ -4,7 +4,7 @@ CONFIG=$1
 GPUS=$2
 NNODES=${NNODES:-1}
 NODE_RANK=${NODE_RANK:-0}
-PORT=${PORT:-29500}
+PORT=${PORT:-29800}
```

Comment:
Better do not make such changes.

Reply:
I should have not committed this file. Reverted.